ui-templates/smoldyn_data.json

```diff
@@ -0,0 +1,56 @@
+{
+    "smoldyn_data": {
+        "python::module": "simulariumio.smoldyn",
+        "python::object": "SmoldynData",
+        "parameters": {
+            "file_contents": {
+                "name": "File",
+                "data_type": "file",
+                "description": "A Smoldyn file",
+                "required": true
+            },
+            "meta_data": {
+                "name": "Metadata",
+                "data_type": "meta_data",
+                "description": "Provide metadata for this trajectory",
+                "required": false,
+                "help": "If not provided, defaults to box size = [100, 100, 100], scale = 1.0, and no metadata about the trajectory or model"
+            },
+            "display_data": {
+                "name": "Display data",
+                "data_type": "collection",
+                "description": "Provide display information for each particle type from Smoldyn",
+                "required": false,
+                "help": "If not provided, defaults to display name from Smoldyn, and draws agent as a sphere with radius = 1.0 and default colors",
+                "length": 1,
+                "extendible": true,
+                "key_item": {
+                    "name": "Particle name",
+                    "data_type": "text",
```

Comment:
are there collections that ever have keys that are not text?
Also, isn't this display data a very standard form that will show up everywhere?  this whole collection thing could become a custom-type, right?